recipes/glib/all/conanfile.py

```diff
@@ -89,90 +89,104 @@ def requirements(self):
             # for Linux, gettext is provided by libc
             self.requires("libgettext/0.21")
 
-        if tools.is_apple_os(self.settings.os):
+        if is_apple_os(self):
             self.requires("libiconv/1.17")
 
     def validate(self):
-        if hasattr(self, 'settings_build') and cross_building(self, skip_x64_x86=True):
-            raise ConanInvalidConfiguration("Cross-building not implemented")
-        if scm.Version(self.version) >= "2.69.0" and not self.options.with_pcre:
+        if Version(self.version) >= "2.69.0" and not self.options.with_pcre:
```

Comment:
Should the `self.info.` versions of `settings` / `options` be used here?

Reply:
no idea, sorry